Add Paul Hudson and Swift Trends to the list

nemanjavlahovic opened the pull request:
I'll add more resources later. Great idea!

nemanjavlahovic committed:
Add Paul Hudson and Swift Trends to the list

carolanitz commented:
great additions! Can you split the commit into two separate ones ? I definitely want to add twostraws but I'm not sure if blogs is the right category for swiftgit.

nemanjavlahovic committed:
Update README.md

Added Erica Sadun, Marin Todorov & Krzysztof Zabłocki to the list. Removed Swift Trends; at least until we figure right section for it.

nemanjavlahovic commented:
Is this ok? Haven't contributed in a while to OSS, so pardon if I messed something up :)

carolanitz commented:
you didn't mess anything up! These are all great additions thank you 👍 . It's just easier to have a PR for each suggestion as mentioned in Contributing.md. For next time ;)
